articles/multi-factor-authentication/multi-factor-authentication-versions-plans.md

```diff
@@ -44,9 +44,9 @@ The following table provides a list of the features that are available in the va
 > This comparison table discusses the features that are part of each version of Multi-Factor Authentication. If you have the full Azure Multi-Factor Authentication service, some features may not be available depending on whether you use [MFA in the cloud or MFA on-premises](multi-factor-authentication-get-started.md).
 
 
-| Feature | Multi-Factor Authentication for Office 365 | Multi-Factor Authentication for Azure Administrators | Azure Multi-Factor Authentication |
+| Feature | Multi-Factor Authentication for Office 365 | Multi-Factor Authentication for Azure AD Administrators | Azure Multi-Factor Authentication |
 | --- |:---:|:---:|:---:|
-| Protect admin accounts with MFA |● |● (Global Administrator accounts only) |● |
+| Protect AAD admin accounts with MFA |● |● (AAD Global Administrator accounts only) |● |
```

Comment:
We never are allowed to abbreviate Azure Active Directory as AAD it is Azure Active Directory or Azure AD